packages/server/gateway-client/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
```

Comment:
I don't think we need the `gateway-client` folder in general, this part will be handled in our React app right?

Reply:
My bad, it wasn't supposed to be commited.
Actually, there is only the ws-echo to see if your messages are going through - as debug purposes.

I will remove it